db/comdb2.c

```diff
@@ -3756,7 +3758,9 @@ static int init(int argc, char **argv)
 
     /* some dbs have lots of tables and spew on startup.  this just wastes
      * peoples time shunting spew */
-    /*showdbenv(thedb);*/
+    if (getenv("CDB2_SHOW_DBENV")) {
```

Comment:
I'm in favor of getting rid of that trace altogether.  I don't think it was ever helpful.

Reply:
It was helpful to me while debugging this change.  I'm not married to it, but it's nice to have it gated behind the environment variable.

Reply:
Works for me.